source/features/link-to-changelog-file.tsx

```diff
@@ -62,10 +57,8 @@ async function init(): Promise<void | false> {
 		'.subnav-links', // Tag list
 	].join(',');
 
-	const navbar = (await elementReady(releasesOrTagsNavbarSelector, {waitForChildren: false}))!;
-	navbar.classList.remove('flex-1'); // Remove margin-right
-	navbar.classList.add('d-flex'); // Avoid wrapping
-	wrapAll(<div className="d-flex flex-justify-start flex-1" />, navbar, changelogButton);
+	const navbar = (await elementReady(releasesOrTagsNavbarSelector));
```

Comment:
I had dropped the parens because they're not necessary
```suggestion
	const navbar = await elementReady(releasesOrTagsNavbarSelector);
```